tools/py/tests.py

```diff
@@ -58,7 +58,7 @@
     args = parser.parse_args()
     tests = args.tests
     print(test_folder[args.tests])
-    exit_code = pytest.main(["--tb=long", "-ra", str(test_folder[args.tests])])
```

Comment:
I found it very helpful to see the whole error message specifying which files are causing the problem on the github CI. if we use the 'line' option, the error message is very concise and hides most of the helpful information from the workflow log. I suggest we use 'long' as we have experienced many times how github can behave differently from our own computers.

Reply:
Do you think it is really necessary now? 
We faced those problems because we were setting everything up. 
With the current body of tests, it is unlikely to happen again.
In most cases, the problem is rather trivial so having such a long report is quite annoying.
What do you think of keeping it in this way for now, and if we see that it turns difficult to find the problems, we come back to the 'long' output?

Reply:
I agree that hopefully, we will not run into those flaky behaviours later. It's definitely good to know about the option "long" for later cases of regtest/example creation.
I actually don't mind leaving it with "line", at all, it makes the workflow logs more compact. indeed. Let's just note for ourselves the option of "long", in case we see unexpected github glitches.